cmd/traffic/cmd/manager/internal/state/intercept.go

```diff
@@ -392,6 +392,11 @@ func (s *State) waitForAgent(ctx context.Context, name, namespace string, failed
 				case "Unhealthy":
 					// Let readiness probe continue, this isn't fatal
 					continue
+				case "FailedMount":
```

Comment:
this might be worth logging at the debug level

Reply:
It is logged by the producer of the event.